keyboards/mio/readme.md

```diff
@@ -0,0 +1,19 @@
+# MIO
+
+![MIO](https://keys.recompile.net/images/mio-01@600w.jpg)
+
+High-end custom gaming keyboard.
+
+* Keyboard Maintainer: [Naoto Takai](https://github.com/takai)
+* Hardware Supported: Original MIO PCB
+* Hardware Availability:  https://keys.recompile.net/projects/mio/
+
+Make example for this keyboard (after setting up your build environment):
+
+    make mio:default
+
+Flashing example for this keyboard:
+
+    make mio:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Could you include instructions on how to enter the bootloader?

Reply:
Thank you! Updated 9af05b1de9627ac3198bcd23ae4fa8d53c6665a3